src/utils/selections.js

```diff
@@ -0,0 +1,60 @@
+import { getAllTreeElemNo } from './tree-utils';
+
+export const selectionState = {
+  linked: false,
+};
```

Comment:
won't this become a shared state between org charts on the same sheet?

Reply:
Yes probably, however sending down linked through the layers won't work. Only when things re-render the new state would then be passed down.

Reply:
Put this on the selectionsAPI now as well